### ISSUE TITLE
OpenXLab model details

### ISSUE THREAD

**Issue by Feynman1999** [sunny2109/SAFMN#14]
whhich model do you upload to OpenXlab? can you share the training details?

**Comment by sunny2109** [comment]
Hi, thank you for your interest.
We provide x2 and x4 pre-trained models for the OpenXlab demo.

For the training detail, we adopt a multi-stage strategy, which uses different loss functions and learning rate in each stage.
The main used loss functions include L1 Loss, FFT Loss,  Perceptual Loss, LDL, and Cosine Similarity Loss.

**Comment by Feynman1999** [comment]
> Hi, thank you for your interest. We provide x2 and x4 pre-trained models for the OpenXlab demo.
> 
> For the training detail, we adopt a multi-stage strategy, which uses different loss functions and learning rate in each stage. The main used loss functions include L1 Loss, FFT Loss, Perceptual Loss, LDL, and Cosine Similarity Loss.

thanks, the training config is not in the repo, right?

**Comment by sunny2109** [comment]
> > Hi, thank you for your interest. We provide x2 and x4 pre-trained models for the OpenXlab demo.
> > For the training detail, we adopt a multi-stage strategy, which uses different loss functions and learning rate in each stage. The main used loss functions include L1 Loss, FFT Loss, Perceptual Loss, LDL, and Cosine Similarity Loss.
> 
> thanks, the training config is not in the repo, right?

Yeah, the whole training procedure is similar to the Real-ESRGAN and you can refer to it for implementing.

**Comment by Feynman1999** [comment]
> > > Hi, thank you for your interest. We provide x2 and x4 pre-trained models for the OpenXlab demo.
> > > For the training detail, we adopt a multi-stage strategy, which uses different loss functions and learning rate in each stage. The main used loss functions include L1 Loss, FFT Loss, Perceptual Loss, LDL, and Cosine Similarity Loss.
> > 
> > 
> > thanks, the training config is not in the repo, right?
> 
> Yeah, the whole training procedure is similar to the Real-ESRGAN and you can refer to it for implementation.

ok， you use the same config (e.g. kernel，resize, blur probability) as  Real-ESRGAN? I want to retrain by myself

**Comment by sunny2109** [comment]
> > > > Hi, thank you for your interest. We provide x2 and x4 pre-trained models for the OpenXlab demo.
> > > > For the training detail, we adopt a multi-stage strategy, which uses different loss functions and learning rate in each stage. The main used loss functions include L1 Loss, FFT Loss, Perceptual Loss, LDL, and Cosine Similarity Loss.
> > > 
> > > 
> > > thanks, the training config is not in the repo, right?
> > 
> > 
> > Yeah, the whole training procedure is similar to the Real-ESRGAN and you can refer to it for implementation.
> 
> ok， you use the same config (e.g. kernel，resize, blur probability) as Real-ESRGAN? I want to retrain by myself

要不我们加个微信(15287108454)，我直接发给你？

**Comment by harusaju** [comment]
Hi, Could you provide the training confs for the SAFMN_Real_x4.pth model ?  I would like to train the model on real world data.
